Xi/extinit.c

```diff
@@ -148,7 +148,6 @@ const Mask DeviceButtonGrabMask = (1L << 17);
 const Mask DeviceOwnerGrabButtonMask = (1L << 17);
 const Mask DevicePresenceNotifyMask = (1L << 18);
 const Mask DevicePropertyNotifyMask = (1L << 19);
-const Mask XIAllMasks = (1L << 20) - 1;
```

Comment:
Hmm, I actually like the old version more. It makes sense to have it defined together with other masks, otherwise it’s too easy to forget about `XIAllMasks`, if a new DeviceProperty*Mask is added or removed. And it’s immediately clear why it’s 20 there when we have other definitions nearby.